qiskit/circuit/quantumcircuit.py

```diff
@@ -2053,7 +2053,8 @@ def id(self, qubit):  # pylint: disable=invalid-name
 
     def ms(self, theta, qubits):  # pylint: disable=invalid-name
         """Apply :class:`~qiskit.circuit.library.MSGate`."""
-        from .library.standard_gates.ms import MSGate
+        # pylint: disable=cyclic-import
```

Comment:
```suggestion
```
Is this still needed (or can it be resolved by changing the below import to `from .library.generalized_gates.gms import MSGate`?